backend.native/tests/iosLauncher/KonanTestLauncher.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,381 @@
+// !$*UTF8*$!
```

Comment:
May be add a `.gitignore` to the Xcode project to exclude user data? See samples: https://github.com/JetBrains/kotlin-native/blob/master/samples/uikit/.gitignore.

Reply:
This is a template project that should not be opened there. It rather should be opened in `./test.output/launcher` as it's a ready for the build there.

Reply:
But what if we want to change something in this template project? Editing a `pbxproj` file is complicated and manual filtering userdata from committed files is boring.